src/FixedDataTableCell.js

```diff
@@ -233,62 +193,48 @@ class FixedDataTableCell extends React.Component {
         'public/fixedDataTableCell/alignRight': props.align === 'right',
         'public/fixedDataTableCell/highlighted': props.highlighted,
         'public/fixedDataTableCell/main': true,
-        'public/fixedDataTableCell/hasReorderHandle': !!props.onColumnReorder,
-        'public/fixedDataTableCell/reordering': this.state.isReorderingThisColumn,
       }),
-      props.className,
+      props.className
     );
 
-    var columnResizerComponent;
-    if (props.onColumnResize) {
-      var columnResizerStyle = {
-        height
-      };
-      columnResizerComponent = (
-        <div
-          className={cx('fixedDataTableCellLayout/columnResizerContainer')}
-          style={columnResizerStyle}
-          onMouseDown={this._onColumnResizerMouseDown}
-          onTouchStart={this.props.touchEnabled ? this._onColumnResizerMouseDown : null}
-          onTouchEnd={this.props.touchEnabled ? this._suppressEvent : null}
-          onTouchMove={this.props.touchEnabled ? this._suppressEvent : null}>
-          <div
-            className={joinClasses(
-              cx('fixedDataTableCellLayout/columnResizerKnob'),
-              cx('public/fixedDataTableCell/columnResizerKnob'),
-            )}
-            style={columnResizerStyle}
-          />
-        </div>
-      );
-    }
-
-    var columnReorderComponent;
-    if (props.onColumnReorder) { //header row
-      columnReorderComponent = (
-        <FixedDataTableColumnReorderHandle
-          columnKey={this.columnKey}
-          touchEnabled={this.props.touchEnabled}
-          onMouseDown={this._onColumnReorderMouseDown}
-          onTouchStart={this._onColumnReorderMouseDown}
-          height={height}
-          {...this.props}
-        />
-      );
-    }
-
     var cellProps = {
       columnKey,
       height,
-      width
+      width,
     };
+    if (this.props.isHeader) {
+      cellProps = {
+        ...cellProps,
+        left: this.props.left,
+        isFixed: this.props.isFixed,
+        scrollToX: this.props.scrollToX,
+        getCellGroupWidth: this.props.getCellGroupWidth,
```

Comment:
why is this a function/getter rather than a normal value prop?

Reply:
These props are not always required and only required while reordering. So, we are prefering to access these only when they are required and not we want to avoid passing unnecesary props which migh affect performance of plugins (We discussed earlier to change it to function instead of directly passing the props)